### PR TITLE
Added the LOFAR link in the README.md file and script

### PR DESCRIPTION
**Pull request by glarocca**
# Summary

---

**Related issue :**

### CODE REVIEW

File: libs/CTA/README.md [hunk 1]
```diff
@@ -1,29 +1,34 @@
...
 * Basic knowledge of the `json`, `csv` and `os` python libraries are requested
 * Python v3.5.2+
 
 ## Usage
+
 * Navigate the [CTA library](https://www.cta-observatory.org/science/library/)
 * Download the HTML web site
-<pre>
+
+  <pre>
```
Comment (gwarf) [inline libs/CTA/README.md]:
```suggestion
```sh
```

File: libs/CTA/README.md [hunk 2]
```diff
@@ -1,29 +1,34 @@
...
 ]$ wget https://www.cta-observatory.org/science/library/
 </pre>
```
Comment (gwarf) [inline libs/CTA/README.md:21]:
```suggestion
```
```

Comment (gwarf) [inline libs/CTA/README.md:21]:
```suggestion
```
```

File: libs/CTA/README.md (hunk 1 above)
Comment (gwarf) [reply libs/CTA/README.md]: @glarocca please use fenced blocks instead of HTML tags, like :
```mardkown
```sh
```

Instead of `<pre>` and `</pre>`

**Review by glarocca — COMMENTED** [review]
approve

**Review by gwarf — APPROVED** [review]
Fine with me.

**Pull request merged**